test/math/ExpDecayMath.ts

```diff
@@ -33,7 +33,7 @@ describe('ExpDecayMath', () => {
                 });
             } else {
                 await expect(expDecayMath.calcExpDecay(ethAmount, timeElapsed, halfLife)).to.revertedWithError(
-                    'Overflow'
+                    'panic code 0x11'
```

Comment:
Should be `revertedWithPanic`

Reply:
I'll give it a try, TX!

Reply:
`Error: Invalid Chai property: revertedWithPanic`

Reply:
Check the documentation: https://hardhat.org/hardhat-chai-matchers/docs/overview: you need to use @nomicfoundation/hardhat-chai-matchers instead of @nomiclabs/hardhat-waffle.

If this is too complicated, then leave it as is.